Pollo/Models/Socket.swift

```diff
@@ -31,26 +32,35 @@ class Socket {
         #endif
         guard let url = URL(string: urlString) else { fatalError("Bad url") }
         guard let accessToken = User.userSession?.accessToken else { fatalError("No access token") }
-        manager = SocketManager(socketURL: url, config: [.log(true), .reconnects(false), .compress, .connectParams([RequestKeys.accessTokenKey: accessToken])])
+        manager = SocketManager(socketURL: url, config: [.log(true), .reconnects(true), .reconnectAttempts(3), .reconnectWait(2), .reconnectWaitMax(10), .compress, .connectParams([RequestKeys.accessTokenKey: accessToken])])
         
         socket = manager.socket(forNamespace: "/\(id)")
         
         socket.on(clientEvent: .connect) { _, _ in
+            let banner = NotificationBanner.connectedBanner()
+            BannerController.shared.show(banner)
+
             self.delegate?.sessionConnected()
         }
         
         socket.on(clientEvent: .disconnect) { _, _ in
+            let banner = NotificationBanner.disconnectedBanner()
+            banner.onTap = { [weak self] in
+                guard let `self` = self else { return }
```

Comment:
dont think you need the back ticks anymore?

Reply:
you right, was just being consistent with the rest of the app lol